server/src/main/java/io/druid/server/security/AuthConfig.java

```diff
@@ -22,37 +22,90 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
+import java.util.List;
+
 public class AuthConfig
 {
   /**
-   * Use this String as the attribute name for the request attribute to pass {@link AuthorizationInfo}
+   * Use this String as the attribute name for the request attribute to pass an authentication token
    * from the servlet filter to the jersey resource
-   * */
+   */
   public static final String DRUID_AUTH_TOKEN = "Druid-Auth-Token";
```

Comment:
It looks like "auth token", "identity", and "user" are used sort of interchangeably. Suggest using the word "identity" for all of them (except JDBC "user" which should still be called "user", as that is a JDBC term).

Also consider using an "AuthenticationResult" object to store two things: the identity string, and the name of the authorizer that should be used to authorize it. This will bundle together the two important things and, I think, make a lot of the code cleaner.

So if you do those things then this should be called `Druid-Authentication-Result` and should contain an AuthenticationResult object. And Druid-Auth-Namespace shouldn't exist, since that's bundled up in the AuthenticationResult.

Reply:
Made `identity` and `AuthenticationResult` the standard names for these concepts